src/test/resources/features/search.feature

```diff
@@ -28,4 +28,18 @@ Feature: Sample
   Scenario: Should be able to navigate to the login page (screenplay)
     Given John is viewing the Etsy landing page (screenplay)
     When he clicks on the sign-in button (screenplay)
-    Then the login page should be presented (screenplay)
\ No newline at end of file
+    Then the login page should be presented (screenplay)
+
```

Comment:
I only added the below scenarios as a bit of fun, and to look more into the Page Object Model methodology as used by Serenity. It should probably not live with the actual interview exercise, but might be useful for our interview discussions.